css/main.css

```diff
@@ -325,3 +325,74 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+  background: #f4f5f7;
+  border-top-left-radius: 50px;
+  padding: 24px;
+  padding-top: 20%;
+  padding-bottom: 20%;
+  gap: 10%;
+  height: auto;
+  align-items: center;
+  text-align: center;
+}
+
+.contact-me h2 {
+  color: #a04300;
+  font-weight: 600;
+  text-align: center;
+  font-size: 40px;
+}
+
+.contact-me .text-me textarea {
+  font-size: 16px;
+  margin-top: 1px;
+  padding: 8px;
+  width: 290px;
+  height: 160px;
+  border-radius: 12px;
+  border-style: none;
+}
+
+.contact-me .text-me .touch {
+  color: #a04300;
+  background: #fff;
+  font-weight: 400;
+  font-size: 16px;
+  width: fit-content;
+  padding: 14px;
+  border-radius: 12px;
+  border-color: #a04300;
+  align-self: center;
+}
+
+.contact-me .text-me .touch:hover {
+  color: #babfed;
+  transform: translate(0, -2px);
+}
+
+.contact-me .text-me {
+  display: flex;
+  gap: 20px;
+  flex-direction: column;
+  justify-items: center;
+  align-self: center;
+  align-content: center;
+}
+
+.contact-me .text-me input {
+  font-size: 16px;
+  padding: 8px;
+  width: 290px;
+  height: 40px;
+  border-radius: 12px;
+  border-style: none;
+}
+
+.submit {
+  width: 500px;
+  height: 50px;
+}
```

Comment:
Great job styling your contact section 😄. I would like to kindly suggest that you stick to the design on the Figma template as much as possible. 

- I believe that the submit button on the template has a background color orange and all input fields have a border-bottom alone.

Please, kindly consider re-styling by taking these suggestions into consideration. 🍷 

## Figma template(Fig1) and Yours (Fig2)
![figform](https://user-images.githubusercontent.com/99246359/197250745-78d22360-b06e-4b54-bc56-a27ece33bf7f.PNG)  ![turiform](https://user-images.githubusercontent.com/99246359/197250823-d173c7af-bb86-4ad8-86b6-c2a86ea9e505.PNG)

Reply:
Thank you @Favourezeugwa for reviewing my project